pinecone/index_connection_test.go

```diff
@@ -269,8 +269,34 @@ func (ts *IntegrationTests) TestUpdateVectorSparseValues() error {
 	actualSparseValues := vector.Vectors[ts.vectorIds[0]].SparseValues.Values
 
 	assert.ElementsMatch(ts.T(), expectedSparseValues.Values, actualSparseValues, "Sparse values do not match")
+}
+
+func (ts *IntegrationTests) TestImportFlow() {
```

Comment:
Might want to do some negative examples (like confirming errors are thrown) too

Reply:
I confirmed that we're returning an error when trying to call `StartImport` with an empty URI.